horizon_bsn/content/connections/reachability_tests/tables.py

```diff
@@ -78,7 +78,11 @@ class RunTest(tables.BatchAction):
     classes = ("btn-edit", )
 
     def action(self, request, id):
-        neutron.reachabilitytest_update(request, id, **{'run_test': True})
+        try:
```

Comment:
can the try-except in api.py be removed if we have a try-except here?

Reply:
Possible but not sure, need to test it..

Since the api can be called via CLI (though not officially supported), I feel we should remove the one here if we want to remove duplicate.

1/2~2/3 of the try catch blocks were added because I couldn't get the error messages when I was testing, the rest were added in the same source file as I expected similar problems for them, though I didn't encounter errors with them.